### ISSUE TITLE
usage message should show AI options

### ISSUE THREAD

**Issue by medovina** [minception/Evora#14]
If I run azul_console with illegal arguments, it displays a usage message:

```
Usage : evora <ai name> <ai name> <number of games>
Available AIs:
	GreedyAI
	MinimaxAI
	MonteCarloAI
	RandomAI
	StrategyAI
```

This message should also show the option syntax and the options that are available for each AI.  The exact format is up to you, but it could look something like this:

```
Usage : evora <ai name> [<option>=<value> ...] <ai name> [<option>=<value> ...] <number of games>
Available AIs:
	GreedyAI
	MinimaxAI [options: time (int), depth (int), abpruning (true or false)]
	...
```

**Comment by medovina** [comment]
Also, please document the default values of all the options somewhere - at the moment the only way to figure out the default values is to read your code.  You could specify the defaults in the usage message, or in 'User guide.odt', or in a manual page - that is up to you.

**Comment by minception** [comment]
fixed in 03d4acc
